packages/components/src/engine/metadata.ts

```diff
@@ -111,6 +111,12 @@ export const getMetadata = (props: IsomerPageSchemaType) => {
   }
 }
 
+export const shouldBlockIndexing = (
+  environment: IsomerPageSchemaType["site"]["environment"],
+): boolean => {
+  return environment === "staging"
+}
+
```

Comment:
shuold we make this more restrictive? we might hvae environments that are non-production which shouldn't be indexed also like `vapt` or `uat`

Reply:
right now we only support `production` / `staging` (generated from our infra)

![image](https://github.com/user-attachments/assets/1d272d6f-f112-4dc0-9114-dffcc81877b3)

we can make it more generic to return true if it **DOES NOT** equate to `production` since we don't know how many non-production sites are there (could have more than staging, vapt, uat in the future too)

WDYT?

Reply:
yea sgtm!

Reply:
updated in https://github.com/opengovsg/isomer/pull/987/commits/e810697509aba6b7547bf72e7408d3f126de1102, thanks!